spec/unit/provider/package/rpm_spec.rb

```diff
@@ -7,12 +7,13 @@
 
   let (:packages) do
     <<-RPM_OUTPUT
-    cracklib-dicts 0 2.8.9 3.3 x86_64
+    'cracklib-dicts 0 2.8.9 3.3 x86_64
```

Comment:
I'd have to get time to pry again and look at instances, I recall line having the same output style but I don't think I put it anywhere